docs/proposals/cli/README.md

```diff
@@ -0,0 +1,143 @@
+# orkactl
+
+## Requirements
+
+For the v0 of orka, only some services are available, which you can see below:
+
+### Provided services
+
+- single tenant  
+- single node  
+- single workload kind (containers)  
+
+
+### Out of scope services (for v0)
+
+- Ressource limitation  
+- Multi tenant
+- Multi workload (VM, containers, bare metal)
+- Scalabilty
+- High Availability
+- Upgradable
+- Monitoring
+- Per tenant networks
+
+
+## High Level Architecture
+
+We are the CLI. As such, we will only communicate with the Cluster API and get information from it.
```

Comment:
I suggest making it less personal. You're not a CLI, afair ;-)

```suggestion
As the cluster CLI, `orkactl` communicates with the Cluster REST API to send requests to it and retrieve cluster information back from it.
```

Reply:
> I suggest making it less personal. You're not a CLI, afair ;-)

*You never know*
![pixiz-11-07-2023-15-33-05](https://github.com/dev-sys-do/orka/assets/24864674/9ea247f5-6b5c-4548-97bf-fdf3673da9ac)

Jokes aside, you're right. The document should be redacted in a more impersonal tone.